scripts/install_dev_tools.sh

```diff
@@ -16,3 +16,8 @@ echo "poetry: $(poetry --version)"
 poetry install
 poetry lock --check
 npm ci
+
+# install cairo-rs-py
+git clone git@github.com:lambdaclass/cairo-rs-py.git
```

Comment:
Also, our current CI runner is probably not sufficient for the new additions. There are several workflows (defined in `.circleci/config.yml`), but e.g. the testing workflow uses `cimg/python:3.8-node` - I believe that will also have to be addressed.

Reply:
Yes, that is exactly the case. With maturin a Linux system with python 3.8 or higher should work just fine.